pkg/cmd/image/list.go

```diff
@@ -245,6 +246,7 @@ func (x *imagePrinter) printImageSinglePlatform(ctx context.Context, img images.
 		Tag:          tag,
 		Name:         img.Name,
 		Size:         progress.Bytes(size).String(),
+		VirtualSize:  progress.Bytes(size).String(),
```

Comment:
Needs comment lines to explain why these are same

Reply:
Also needs integration tests

Reply:
yep! Wanted to get an initial look. Will add by the end of this week.
Thanks!